avo-hq/avo
# fix: default filters cleared on apply

# Description


Fixes https://github.com/avo-hq/avo/issues/841

# Checklist:



## Manual review steps


1. Go to the users resource
2. open the filters panel
3. observer the "Is admin" filter to have both elements selected
4. write something in the other text field and filter by that
5. when the page reload observe the "Is admin" filter kept the defaults
